frontend/src/components/doctors/components/filtration/filtration.tsx

```diff
@@ -53,22 +62,28 @@ const Filtration: React.FC = () => {
               errors={errors}
             />
           </Details>
-          <Details icon={Icon.SPECIALTY} title="Specialty">
-            {doctorSpecialties.map((doctorSpecialty) => (
+          <Details
+            icon={Icon.SPECIALTY}
+            title="Specialty"
+          >
+            {doctorSpecialties.map((doctorSpecialty, index) => (
               <div className={styles.filterCheckbox} key={doctorSpecialty}>
                 <Checkbox
-                  name={doctorSpecialty}
+                  name={DoctorFiltration.SPECIALTY + `[${index}]`}
                   label={doctorSpecialty}
                   control={control}
                 />
               </div>
             ))}
           </Details>
-          <Details icon={Icon.CLINIC} title="Type of clinic">
-            {clinicTypes.map((clinicType) => (
+          <Details
+            icon={Icon.CLINIC}
+            title="Type of clinic"
+          >
+            {clinicTypes.map((clinicType, index) => (
```

Comment:
I think that now we have labels in `lowercase`. It should be the same as in the design
![image](https://user-images.githubusercontent.com/60396171/111742926-92d4ff00-8891-11eb-8646-fe7bc5f9dd4a.png)